src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,79 @@
+.contacts {
+  &__content {
+    @include on-tablet {
+      display: flex;
+      gap: 50px;
+    }
+
+    @include on-desktop {
+      display: flex;
+      gap: 50px;
+    }
+  }
+
+  &__info {
+    width: 100%;
+    gap: 50px;
+
+    @include on-tablet {
+      display: block;
+    }
+
+    @include on-desktop {
+      display: flex;
+      gap: 50px;
+    }
+  }
+  &__form {
+    width: 100%;
+  }
+
+  &__link {
+    text-decoration: none;
+    font-weight: 700;
+    color: $main-text-color;
+
+    &:hover {
+      color: $accent-bg-color;
+    }
+  }
+
+  &__label {
+    font-weight: 400;
+  }
+}
+
+.form {
+  margin-bottom: 84px;
+
+  &__name {
+    @include form-fields;
+
+    height: 48px;
+  }
+  &__email {
+    @include form-fields;
+
+    height: 48px;
+  }
+  &__message {
+    @include form-fields;
+
+    height: 144px;
+  }
+  &__button {
+    width: 100%;
+    height: 56px;
+    margin-bottom: 16px;
+    border: none;
+
+    font-family: Cinzel, 'Times New Roman', serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 27px;
+    text-align: center;
+
+    color: $accent-text-color;
+    background-color: $accent-bg-color;
+  }
```

Comment:
Your form button is missing hover effects to indicate the element can be interacted with [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Hightlight%20clickable%20element%20with%20cursor%3A%20pointer%20and%20some%20%3Ahover%20styles)

Reply:
Done.